packages/runtime/container-runtime/src/opLifecycle/batchManager.ts

```diff
@@ -54,9 +55,14 @@ export class BatchManager {
 
 	constructor(public readonly options: IBatchManagerOptions) {}
 
-	public push(message: BatchMessage, currentClientSequenceNumber?: number): boolean {
+	public push(
+		message: BatchMessage,
+		currentClientSequenceNumber?: number,
+		reentrant?: boolean,
```

Comment:
```suggestion
		reentrant: boolean = false,
```
To help simplify conditional below